From f8d542d2cb0eb4f9ceb583cde84fed99ede42e34 Mon Sep 17 00:00:00 2001
From: dianlujitao <dianlujitao@gmail.com>
Date: Thu, 10 Jul 2014 18:56:16 +0800
Subject: [PATCH] Don't use MSM_BSP for msm7x27a

---
 minui/Android.mk | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/minui/Android.mk b/minui/Android.mk
index 48febf4..2841579 100644
--- a/minui/Android.mk
+++ b/minui/Android.mk
@@ -21,8 +21,10 @@ LOCAL_C_INCLUDES +=\
 #  LOCAL_C_INCLUDES += $(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr/include
 #endif
 
-ifeq ($(TARGET_USES_QCOM_BSP), true)
-    LOCAL_CFLAGS += -DMSM_BSP
+ifneq ($(TARGET_BOARD_PLATFORM),msm7x27a)
+  ifeq ($(TARGET_USES_QCOM_BSP), true)
+      LOCAL_CFLAGS += -DMSM_BSP
+  endif
 endif
 
 LOCAL_MODULE := libminui
